modules/cluster-logging-collecting-and-forwarding-logs-only.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-collector.adoc
+
+[id="cluster-logging-collecting-and-forwarding-logs-only_{context}"]
+= Collecting and forwarding logs only
+
+As an administrator, you can collect and forward logs to an external log store and minimize resource consumption by eliminating other components.
```

Comment:
CLF can forward logs to multiple destinations and they are not necessarily external to the OpenShift cluster.
Consumption of OpenShift cluster resources can be a reason to do it, if the destinations are off-cluster, but there can be other reasons as well, like secure storage of audit logs or use of an enterprise-wide logging store.

Reply:
Updated. Thanks!